tests/pipelines/stable_diffusion/test_stable_diffusion_pix2pix_zero.py

```diff
@@ -539,47 +539,6 @@ def test_stable_diffusion_2_pix2pix_inversion(self):
 
         assert np.abs(expected_slice - image_slice.cpu().numpy()).max() < 5e-2
 
-    def test_stable_diffusion_pix2pix_full(self):
```

Comment:
Why are we getting rid of it entirely?

Reply:
The pipeline wasn't prioritised in [this list](https://github.com/huggingface/diffusers/issues/4527#issue-1840665874) (under highly important and important) 

So I moved all the slow tests to nightly.